docs/cbenv.adoc

```diff
@@ -2,24 +2,26 @@
 == `cb-env` and the Environment
 
 Whilst multiple clusters can be registered at the same time, there is only ever one cluster (at most) active.
-The same is true for buckets, scopes, and collections.
-When a resource is active then it used as the default to run commands against (this can be overridden on a per command basis).
-
-You can run the `cb-env` command, which will tell you which resources are currently active (you are also able to tell from the prompt):
+The same is true for buckets, scopes, and collections. When a resource is active it is used as the default to run
```

Comment:
Generally try for a single line per sentence, which I believe is what the docs team do.

Reply:
Done!